src/main/java/com/iopipe/IOpipeConfiguration.java

```diff
@@ -142,11 +150,19 @@
 			throw new IllegalArgumentException("The timeout window cannot " +
 				"be negative.");
 		
+		if (serviceurl == null)
```

Comment:
What's the scenario in which this occurs? I would say set serviceurl (serviceUrl?) to a default, which is IOpipe

Reply:
This would only occur if someone is manually setting up the service and they forgot to set the service URL, but I can have it fallback to a default address if one is not specified instead of failing.